src/messages.js

```diff
@@ -59,7 +59,7 @@ const messages = {
   onboardingSuccessful: '🎉 You are now successfully onboarded !!! 🎉',
   alreadyOnboarded: (role) => `✅ Onboarded as ${role}`,
   networkSelected: (network) => `Network: ${network}`,
-  healthCheckUrl: `After starting your node, you can check its health by going to Explorer: https://explorer.ambrosus.com/`,
+  healthCheckUrl: () => `After starting your node, you can check its health by going to Explorer: https://explorer.ambrosus.com/`,
```

Comment:
Why no parameter?

Reply:
From previous PR. @seagiv changed with no parameters. Just to print explorer url.

Reply:
But why print explorer URL, while this is `healthCheckUrl` message, used in `healthCheckUrlDialog`, which is used to display node's healthCheck endpoint URL.